### DISCUSSION TITLE
Backup config

### DISCUSSION THREAD

**Discussion by madprops** [Taiko2k/Tauon#1197] [General]
If I wanted to backup Tauon's config to github, what could be a good way?
I tried looking into: /home/user/.var/app/com.github.taiko2k.tauonmb/config/
But that doesn't seem to be it

**Comment by Taiko2k** [comment]
It might be under the data folder, its called `tauon.conf`, but it might not be that useful to back that up as not all settings are stored there. Many are stored in the `state.p` file which is a large binary file.

**Comment by madprops** [comment]
Is the binary holding song data? I'm mostly interested in the settings I customize through the settings window and others, so I don't have to re-do it again if I have to reinstall Tauon.

**Reply by Taiko2k** [reply]
Yes the binary file containes Tauons song and playlist database. So that may contain personal information so I wouldn't back that up. But you can backup `tauon.conf` to preserve *some* settings.

**Reply by madprops** [reply]
Ok. I noticed `tauon.conf` also contains passwords, so I'd have to be careful.
